apps/web/src/studio/studio.types.ts

```diff
@@ -19,3 +19,16 @@ export interface IBridgeWorkflow {
   code: string;
   steps: IBridgeWorkflowStep[];
 }
+
+export interface ILocalStudioState {
+  testUser: {
+    id: string;
+    firstName: string;
+    lastName: string;
+    emailAddress: string;
+    profilePicture: string;
+  };
```

Comment:
```suggestion

export interface ILocalStudioUser {
    id: string;
    firstName: string;
    lastName: string;
    emailAddress: string;
    profilePicture: string;
 };
 
export interface ILocalStudioState {
  testUser: ILocalStudioUser;
```

Alternatively, if this is just a subset of actual user info, just use `Pick<UserType, ...>`